src/index.css

```diff
@@ -40,6 +40,27 @@ body {
   padding: 0 1em;
 }
 
+/* AboutPage component */
+.about-page-container{
+  width: 70%;
+  margin: 0 auto;
+}
+
+.about-page-content{
+  margin: 0 auto 2em;
+  display: flex;
+  flex-direction: row;
```

Comment:
Default `flex-direction` is `row`. Not a big deal, just wanted to mention this line could be omitted.

Reply:
Good catch! DRY code is important. Thank you